assets/sparksql/search.py

```
@@ -0,0 +1,19 @@
+# executed within container
+import sys
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)